jenkins/databricks/test.sh

```diff
@@ -20,6 +20,7 @@ set -ex
 LOCAL_JAR_PATH=${LOCAL_JAR_PATH:-''}
 SPARK_CONF=${SPARK_CONF:-''}
 BASE_SPARK_VER=${BASE_SPARK_VER:-'3.0.1'}
+[[ -z $SPARK_SHIM_VER ]] && { SPARK_SHIM_VER=spark${BASE_SPARK_VER//.}db }
```

Comment:
only downside to this is for dev use we didn't specify and it would still run correct tests just skip cache test.  may just need to update our internal docs

Reply:
why would it skip cache tests?

Reply:
I made this 3.1.1 to match build.sh. It won't skip the cache test by default.